Packs/CortexAttackSurfaceManagement/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.5.9",
+    "currentVersion": "1.6.1",
```

```suggestion
    "currentVersion": "1.5.10",
```
We haven't talked about what is revision vs minor version, but typically have been doing minor versions